src/ReadLog.jl

```diff
@@ -0,0 +1,134 @@
+# Read LCM log files directly
+
+# export
+#   LCMlog,
+#   close,
+#   readNextEvent,
+#   handle,
+#   subscribe
+
+# also see JuliaLang's RawFD type, but little information found.
+# www.c4learn/c-programming/c-file-structure-and-file-pointer
+mutable struct FILE
+  level::Cshort
+  token::Cshort
+  bsize::Cshort
+  fd::Cuchar
+  flags::Cuint  # (unsigned flags ;)
+  hold::Cuchar
+  buffer::Ptr{Cuchar}
+  curp::Ptr{Cuchar}
+  istemp::Cuint # (unsigned is temp)
+end
+
+mutable struct lcm_eventlog_t
+  f::Ptr{FILE}
```

Comment:
I think if this is a `Ptr{Void}` then we can lose the whole `FILE` struct

Reply:
Sure, will change back -- I actually started with `Ptr{Void}`. Also not sure if `Base.RawFD` is trying to do the same thing?

Reply:
Ok, thanks! I think RawFD is wrapping a Unix file descriptor, rather than a C file, but I admit I'm not entirely sure.